content/en/blog/2023/http-conventions-declared-stable.md

```diff
@@ -0,0 +1,271 @@
+---
+title: HTTP semantic conventions declared stable
+linkTitle: HTTP semconv are stable
+date: 2023-11-03
+author: '[Trask Stalnaker](https://github.com/trask) (Microsoft)'
+cSpell:ignore: Liudmila Molkova
+---
+
+Early this year we embarked on a three month project to stabilize HTTP semantic
+conventions. We are proud to announce (nine months later), that the HTTP
+semantic conventions are now the first OpenTelemetry semantic convention to be
+declared stable! The stable version is a significant evolution over the previous
+unstable version:
+
```

Comment:
... such as? Would be good to provide a bit more context and an example here.

Reply:
the list below are the examples, I think @chalin's suggestions have made this clearer, but let me know what you think